sunpy/map/tests/test_map_factory.py

```diff
@@ -251,6 +251,22 @@ def test_url_pattern():
     amap = sunpy.map.Map("http://data.sunpy.org/sample-data/AIA20110319_105400_0171.fits")
     assert isinstance(amap, sunpy.map.GenericMap)
 
+@pytest.mark.remote_data
+def test_uri_pattern():
+    """
+    Testing publicly accessible s3 object
+    """
+    amap = sunpy.map.Map("s3://data.sunpy.org/sunpy/AIA20110607_065843_0193_cutout.fits", fsspec_kwargs={"anon": True})
+    assert isinstance(amap, sunpy.map.GenericMap)
+
+@pytest.mark.remote_data
+def test_uri_directory_pattern():
+    """
+    Testing publicly accessible s3 directory
+    """
+    with pytest.warns(SunpyUserWarning, match='Failed to read'):
+        amap = sunpy.map.Map('s3://data.sunpy.org/aiapy', fsspec_kwargs={'anon':True}, allow_errors=True)
```

Comment:
What happened here?

Reply:
What do you mean? The warning is raised by the fact there's a README file in the directory which isn't a valid map.

Reply:
Oh I thought it was a real file. Ignore me.